examples/README.md

```diff
@@ -60,7 +60,7 @@ Example Configuration:
 - `CONFIG_EXAMPLES_AJOYSTICK_DEVNAME` – Joystick device name. Default
   `/dev/adjoy0`.
 - `CONFIG_EXAMPLES_AJOYSTICK_SIGNO` – Signal used to signal the test
-  application. Default `13`.
+  application. Default `30`.
```

Comment:
Now it need to be moved at least to SIGRTMIN (aka 32)

Reply:
Done.